src/tools/rbd/action/BenchWrite.cc

```diff
@@ -262,6 +262,11 @@ int execute(const po::variables_map &vm) {
   } else {
     bench_io_size = 4096;
   }
+  if ((bench_io_size > 100*1024*1024) || (bench_io_size < 1))
```

Comment:
@x11562 Can you provide the crash backtrace?  I can run with the same arguments without any crashes.  There was a bug with readahead and sequential IO that would make it read off the end of the image.